cmd/daemon/main.go

```diff
@@ -1,8 +1,63 @@
 package main
 
-import "fmt"
+import (
+	"context"
+	"log"
+	"net/http"
+
+	authv1 "k8s.io/api/authentication/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	"knative.dev/container-freezer/pkg/daemon"
+	"knative.dev/container-freezer/pkg/freeze/containerd"
+	pkglogging "knative.dev/pkg/logging"
+)
+
+const runtimeTypeContainerd string = "containerd"
 
 func main() {
-	// TODO delete -- temporarily added to merge hack project
-	fmt.Println("hello, freezer!")
+	logger, _ := pkglogging.NewLogger("", "")
```

Comment:
this probably shouldn't be empty strings in production (rather than test code) but should come in via an env var. We can spin out an issue though if you like